lib/ExpandableListItem/ExpandableListItem.less

```diff
@@ -1,11 +1,12 @@
 /* ExpandableListItem Header*/
 .moon-expandable-list-item-header {
+  padding-right: @moon-spotlight-outset + 30px;
```

Comment:
@Djspaceg Yes, some css related with expandablePicker need to rework.
Some controls like ExpandableListItem, Accordion have css dependency issue.